crypto/asn1/a_d2i_fp.c

```diff
@@ -92,7 +91,7 @@ void *ASN1_item_d2i_fp(const ASN1_ITEM *it, FILE *in, void *x)
 
 #define HEADER_SIZE   8
 #define ASN1_CHUNK_INITIAL_SIZE (16 * 1024)
-static int asn1_d2i_read_bio(BIO *in, BUF_MEM **pb)
+int asn1_d2i_read_bio(BIO *in, BUF_MEM **pb)
```

Comment:
Because I needed a function that reads a DER blob, verifies that it's a DER blob, but then gives me that DER blob as is without further interpretation into specific OpenSSL types (that's done by the individual file loader handlers later on).  Essentially, I needed `ASN1_d2i_bio` without the `d2i` part, and that's *exactly* what this function does.

Of course, I could have copied this function into the file loader, but I'm not much a fan of code copying.  Ergo...